### PR TITLE
Big Medicine Rebalance V2.7

### PR DESCRIPTION
**Pull request by Isy232**
## About The Pull Request

See changelog for a full list of the changes.

EVERYTHING IN THIS PR IS SUBJECT TO CHANGE.

## Why It's Good For The Game

Basic chems are far too OP while advanced ones are next to useless. 

This leads to chemists easily being able to produce extremely powerful healing chemicals while the rare and difficult ones are considered useless and never made.

By nerfing the basic ones and buffing the advanced ones chemistry becomes more complex and rewards knowledge more.

However basic chems like bicardine can still get the job done, just less quickly.

## Changelog
:cl: Ruko, Isy232
balance: Styptic powder, Synthflesh and Silver Sulf now do nothing when splashed.
balance: Styptic powder and Silver Sulf now do 0.5 healing per tick and 1 when instantly applied but their OD threshold is upped to 100u and the metabolism rate is increased to 1u per tick. A 40 Styptic powder patch will heal 40 to the limb it is applied to and then 20 damage over 40 ticks. They also do stamina damage equal to the amount applied x2, the medicine is painful so this is pain damage.
balance: Atropine's brute, toxin and fire damage healing is doubled and it only looses its effectiveness at 80 damage instead of critical condition. Atropine also now causes dropping of items in its confusion state so its no longer suitable for combat use and its overdose threshold is lowered to 15u. Overdosinng on atropine now causes atropine to become histamine in a 2 to 3 reaction.
balance: Bicardine, Kelotane, Dexalin and Anti-toxin now heal 1.5 per tick while below 50 damage and 0.5 per tick when above 50 damage. This makes them no longer as effective on large wounds but they are quite capable of healing smaller ones. Their metabolism rate has also been halved so they stick around longer.
balance: Salicyclic acid, Dexalin Plus and Oxandrolone now heal 3 damage per tick. They also do 2 stamina damage per tick up to 80. They now only do stamina damage while healing.
balance: Anti-toxin no longer purges chems and only heals toxin damage. Use charcoal or something.
balance: The overdose of most chems now causes organ damage instead of just reversing the effects of what the chem healed.
add: Adds bicaridine/kelotane pill bottles to burn/brute treatment kits.
balance: Brute and burn patches now contain 30u of their reagents instead of only 20.
balance: Perfluorodecalin now causes more toxin damage as a side effect, overdoses at 30u and causes lung damage upon overdose.
balance: Salbutamol can now cause an overdose. The threshold is 25u (I made salbutamol pills have less u to avoid OD on them). This overdose causes Sulbutamol to become histamine in a 1 to 1 reaction.
balance: Corazone now heals the heart by 1.5 per tick, however it now has an overdose threshold at 20u and if you overdose it will turn into histamine in a 1to 1 reaction.
balance: Carthatoline now heals 3 toxin loss per tick and its OD causes brain damage. Its overdose threshold is now 25u.
balance: Leporazine now has an overdose at 30u. This overdose causes your temperature to rapidly increase or decrease randomly.
add: You can now find bicaridine and kelotane pill bottles in medical vendors.
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
Good change.

**Comment by Chessus** [comment]
#Edit. PA change reverted

I'm for these changes. 
Not 100% on atorpine yet  as I think it should only work in crit as Bacon suggested.

**Comment by Isy232** [comment]
> #Edit. PA change reverted
> 
> I'm for these changes.
> Not 100% on atorpine yet as I think it should only work in crit as Bacon suggested.

I made atropine only work up until 80 damage. So atro will stop working at 20% healthy.

I think this is justified by atropine being probably the most complex healing chemical in the game.

Atropine requires 5 different chemicals to make, 3 of which are complex chemicals. Those 3 complex chemicals are also complex chemicals which require 1 complex chemical to make.

Atropine is the endgame of chemistry for healing, so it should be quite good.

**Comment by AnjelicaChaykovsky** [comment]
> Atropine is the endgame of chemistry for healing, so it should be quite good.

laughs in new robust chem for odysseus

**Comment by Isy232** [comment]
> > Atropine is the endgame of chemistry for healing, so it should be quite good.
> 
> laughs in new robust chem for odysseus

I don't know what you're talking about.

**Comment by eeSPee** [comment]
Overall good changes, but I don't like the synthflesh nerfs. Synthflesh is overnerfed as is. I always thought as synthflesh as the splash-to-heal medicine. I also don't think removing the synthflesh sprays is a good idea. They were quite decent, and a nice supply of synthflesh.

**Comment by Sinmero** [comment]
I dunno about atropine but its a good change overall.

> Synthflesh is overnerfed as is

Currently Synthflesh is the most op chem.

**Comment by AnjelicaChaykovsky** [comment]
> 
> I don't know what you're talking about.

If you know, you know.

**Comment by MrHorizons** [comment]
If you want bicard/kelo to be more powerful for smaller wounds, shouldn't you make it heal faster __below__ 50 damage?

**Comment by Isy232** [comment]
> If you want bicard/kelo to be more powerful for smaller wounds, shouldn't you make it heal faster **below** 50 damage?

Thats what it does do I'm just shit at wording.

**Comment by Chessus** [comment]
Salicylic acid and oxladrone are basically bicarb and keep with stamina loss now?

**Comment by Isy232** [comment]
> Salicylic acid and oxladrone are basically bicarb and keep with stamina loss now?

Well, kinda, they're 1.5x better. The stamina loss is minimal and won't cause issues in small doses.

Actually I got a better idea, I'll make them only do stamina when they're healing.

**Comment by Rukofamicom** [comment]
Isy went to sleep before updating the changelog, but this should be ready to present to the Maintainers tomorrow.

**TL;DR** This PR updates all of the core healing chems by categorizing them into four groups, with each group having it's own strengths and weaknesses. This PR will keep the most accessible medicine useful, but make the higher end medicines worth the effort it takes to make them and reward players for mixing chems together for more potent medicines. This PR also targets the stacking of chems before entering combat to make sure it doesn't just immediately replace splashing.

**Basic chems** (Bicaridine, Kelotane, Anti-toxin, Dexalin, Tricordrizine), are now less effective overall, but remain the basic go-to for beginners and lazy players alike. They will get the job done with no side effects, minimal overdose downside and now metabolize at half the rate they used to so they stay in your system to continue healing for much longer. Except for Tricordrizine, these chems are much slower at healing severe injuries leaving players to choose between waiting out a slow healing process, or getting better chems/care.

**Touch-based chems** (Styptic Powder, Silver Sulfadiazine, Synthflesh) have been nerfed substantially so they are no longer the king of chems, but are instead the "quick and dirty" option for when a greytider on the go needs healing fast. Touch chems are now metabolized very rapidly for low healing making them primarily heal the region they are applied to, with only 1/3 of the healing being applied over time and over the rest of the body. The instant portion of the heal also applies stamina damage to the limb it is applied to (it really, really stings!) and will easily stamcrit a limb, but will only entirely stamcrit a player if a full 50u patch is applied to the chest (49u will not). Overall you get a burst of healing for only about a 15 second downtime even if you do stamcrit. 

**Advanced chems** (Salicylic Acid, Oxandrolone, Carthatoline, Dexalin+) now heal steadily regardless of how much damage the patient has, although the potential healing is lower now (3 instead of 4). Except for Carthatoline, each of these chems will convert the damage they heal into stamina damage. This means they only do stamina damage while healing, but over-reliance on them in combat will result in stamcrit, although the stamcrit will happen on a fairly substantial delay from when they player would have entered crit without the drugs. *It's hard to really describe the implications, better to test and see for yourself*. Carthatoline instead causes vomiting which helps purge other chems from the body, but also periodically stuns.

**Specialized Chems** can't really be summarized because they each do something either unique or relatively unique. Most of these weren't changed much, but they all have an overdose now

**Comment by Rukofamicom** [comment]
With the above post, some example mixtures and their various effects:

**Tricordrizine + Dexalin** - Poor man's Omnizine, will heal just about any injury but will take a while to do it. Apply CPR to speed up the recovery from crit. 

**30u Styptic patch from a medkit** - if applied to a limb, puts the limb into stamcrit. If applied to chest, the player is slowed for a short while. Heals 30 brute instantly, and 15 brute over time. (Once the Stamina PR is done, both stamina effects for a single patch will be negligible, but two in quick succession will leave you in some pain)

**50u patch with 25/25 Salicylic/Oxandrolone** - will give a player rapid regeneration of both brute and burn damage, but will stamcrit the player very quickly if both types of damage are being healed at the same time. 100 total healing = 100 stamina damage; you would have been in real crit without the drugs by this point.

**6u Oxandrolone pill** - Heals 45 burn damage (two red lasers) over 15 ticks, deals 45 stamina damage which isn't enough to slow.  Small, bursty dosage with negligible downside in most situations - with careful timing of stamina regeneration, this can replace the Synthflesh meta with a higher skillset requirement.

Basic chemistry remains accessable and useable with each individual chem offering slightly different effects and effectiveness which pushes players to mix and experiment. Meanwhile careful mixtures and dosages of different chems (especially those including Meth or Arenesp to offset stamina damage) will now take precedence over the zero-effort/knowledge trick that is bottle splashing, and still provide a substantial advantage that rewards the player which bothers to experiment and learn them.

**Comment by Chessus** [comment]
Kinda getting a cobbychem vibe from this.  I was fine with the initial PR that was closed. I don't think this will pass if put to a vote. Only if you incrementally add these in it will pass imho

**Comment by Sinmero** [comment]
> This overdose causes Salbutamol to become histamine in a 2 to 4 reaction.
Corazone now heals the heart by 1.5 per tick, however it now has an overdose threshold at 20u and if you overdose it will turn into histamine in a 2 to 4 reaction.

The od effects are WAY too much. You get at least 40 histamine on corazone od and 50 on sulbutamol od.
Histamine has od threshold at 30u and will deal 2 brute 2 oxyloss and 2 tox damage each tick.

Salbutamol and Corazone should have a 5-15% chance to turn into histamine like venom or formaldehyde.

**Comment by YakiAttaki** [comment]
cobby is that you?

**Comment by St0rmC4st3r** [comment]
> The od effects are WAY too much. You get at least 40 histamine on corazone od and 50 on sulbutamol od.
> Histamine has od threshold at 30u and will deal 2 brute 2 oxyloss and 2 tox damage each tick.
> 

Do you forget the existence of antihistamine medicine?

**Comment by Sinmero** [comment]
> Do you forget the existence of antihistamine medicine?

Histamine has 0.1 metabolism and deals total 6-8 damage/tick when overdosed.
Getting histamine od is lethal and it will kill you fast.
By getting salbutamol/corazone od you get histamine od in 8 ticks. 
You get od just by eating two salbutamol pills (that are available everywhere roundstart).
Getting od on healing chems should not be more lethal than combat toxins.

**Comment by St0rmC4st3r** [comment]
@Isy232 I guess we need that epinephrine use we were talking about before.

**Comment by Isy232** [comment]
Histamine is a traitor toxin you can get in the traitor toxin kit, so I don't want to nerf it too much.

Rather I will just nerf the amount of histamine produced.

**Comment by Rukofamicom** [comment]
I'm not entirely sure how this is comparable to cobbychem :thinking: 

CobbyChems has overdoses that reverse the effects of the medication, just like our current system
IsyChems cause organ damage without reversing the chem effect, so you aren't suddenly taking the damage you desperately need healed. Only specialized, non-standard medicines have more dramatic overdose effects - the kind an average doctor isn't likely to make or seek without already knowing about it. 

CobbyChems has dramatic downsides to a lot of chems, which require secondary medication to treat.
IsyChems cause stamina damage which currently requires a 10 second wait to completely vanish, and after stamina PR will require a 6 second wait to end crit, with only a few more before the slowdown goes away. Perfluorodecalin already has toxin damage on current code without our tweak which allows it to heal lung damage.

The only connection I can draw is because the community wants to keep splashing in its current state of being so overpowered that medbay may as well be a cloning room staffed by a single chemist with just enough plumbing to make a synthflesh factory. If you think keeping an entire department so irrelevant that players *literally groan* when a doctor tries to be a doctor, there's something wrong with how you're approaching the game.

**Comment by WhyIsCaeciliusTaken** [comment]
Or to put it more bluntly:

removing chemistry's ability to generate infinite instant heal to full bottles is not comparable to making chems require a 10 minute long juggling act to heal a single electric shock.

**Comment by Isy232** [comment]
> Kinda getting a cobbychem vibe from this. I was fine with the initial PR that was closed. I don't think this will pass if put to a vote. Only if you incrementally add these in it will pass imho

Bacone1

Again, maintainers get the final say, if they think we nerfed it too hard we'll make another revision until it works.

**Comment by AnjelicaChaykovsky** [comment]
Instead of changing chems you could always go other way around and focus your effort  on changing wound system.
There already was vote on porting the wound system and people were loving it.
Porting tg,s health system would mean that you could develop wounds that can be fixed with only surgery and are slowly killing you or cripple you.

**Comment by Isy232** [comment]
> Instead of changing chems you could always go other way around and focus your effort on changing wound system.
> There already was vote on porting the wound system and people were loving it.
> Porting tg,s health system would mean that you could develop wounds that can be fixed with only surgery and are slowly killing you or cripple you.

The point of this PR is dealing with instaheal beakers and making doctors a critical part of medbay.

Having things like bone breaks that require surgery adds a LOT more issues. For example, what about nukies or other antags that need to fight a lot and don't get much a chance to get full treatment.

**Comment by ghost** [comment]
> Having things like bone breaks that require surgery adds a LOT more issues. For example, what about nukies or other antags that need to fight a lot and don't get much a chance to get full treatment.

Splints.

Paradise has a Traitor item that is an injector that instantly heals bone breaks.

**Comment by AnjelicaChaykovsky** [comment]
> The point of this PR is dealing with instaheal beakers and making doctors a critical part of medbay.

Changing a few digits in a chem doesn't fix either of them, does adding traumas fix this ?
Yes, also it's not just bone breaks, best if you read this if you want to make doctors more important.
https://tgstation13.org/wiki/Guide_to_Wounds

**Comment by ghost** [comment]
> > The point of this PR is dealing with instaheal beakers and making doctors a critical part of medbay.
> 
> Changing a few digits in a chem doesn't fix either of them, does adding traumas fix this ?
> Yes, also it's not just bone breaks, best if you read this if you want to make doctors more important.
> https://tgstation13.org/wiki/Guide_to_Wounds

image
I do want this but uhhh, good luck.

**Comment by AnjelicaChaykovsky** [comment]
> I do want this but uhhh, good luck

Maybe one day someone based enough will port it

**Comment by Rukofamicom** [comment]
> Changing a few digits in a chem doesn't fix either of them

No, you're definitely wrong. Changing a few numbers very easily fixes OP instant healing.

> The point of this PR is dealing with instaheal beakers and making doctors a critical part of medbay.

However, I don't think this will make doctors a critical part, it's just a step in that direction by making doctors less obsolete.

**Comment by Chessus** [comment]
@Rukofamicom I agree, as a medical main I'm well aware. We agree about changes to clonning and insta heal chems. Unfortunately, player consensus has an effect on changes and that's the reality we face until the changes are forced, not voted upon. Big changes scare players and I'm sure that this would fail if put to a vote. 

I was drawing the comparison of having negative effects while taking the chem, it punishes the player which is unnecessary. Chem juggling isn't fun either. I think just decreasing the amount they heal with longer metabolism, give them certain thresholds  or even adding interactions with other chems is a smarter way to go about this. This can make doctoring look appealing if there's **analogs to the chemical**(there isnt surgery for toxin damage or oxygen loss, drinking or radiation). 

A doctor is a healer and administrator chemicals to heal so nerfing chemicals to the point no one wants to use them would just make doctors surgeons. Players circumvent doctors, they cut out the middle man. So perhaps the problem lies in the administration of the chemicals? Plus no player wants to be vulnerable hence why No2 is never used and lying on a surgery bed perhaps without headset/uniform. That's why doctoring is unappealing. 

I agree with the first changes of removing the splash heal. Its a bit much but with the other changes, as a patient I don't want some convoluted healing system. I want to be healed and get back to it. So I don't see why the current arcade-y system is bad, medical can't cant usually keep up with the bodies and with chemists fucking off this makes it even worse. If all of these changes are implemented then this is just going to make those key moments worse. 

*Edit* Until there's an executive decision made like tg does, we either have to cater creative changes to the current meta or risk just grinding our wheels.

**Comment by Isy232** [comment]
I think we should have a vote with 3 options for players to choose.

1: Remove just splash healing
2: Rework medical a bit, but not as heavy of nerfs as there are now
3: I want the current medicine rebalance

I've asked the maintainers about it, I dunno what they'll say

**Comment by Rukofamicom** [comment]
> (there isnt surgery for toxin damage or oxygen loss, drinking or radiation).

There is CPR for oxyloss, and I don't really think any of the rest of these should have a surgery at all - they make sense to be 100% chemically treated.

> A doctor is a healer and administrator chemicals to heal so nerfing chemicals to the point no one wants to use them

@Chessus only the touch based chems are significantly nerfed in current state. All other chems are either straight buffed or significantly more gained than lost. Tricordrizine has been straight buffed, Salicylic and the others in that group have not only been buffed but are now stronger than any current chem in the game at raw healing speed. Atropine has been more than doubled in effectiveness with only a minor drawback.

Saying chems as a whole have been nerfed is extremely misleading.

**Comment by ike709** [comment]
As a medical main I approve this message.

I agree that a much larger medical rework is needed, but until then this is a decent stopgap measure.

**Comment by Chessus** [comment]
> There is CPR for oxyloss, and I don't really think any of the rest of these should have a surgery at all - they make sense to be 100% chemically treated.

@Rukofamicom  I have an idea for an iron lung as joke lol. I meant husking instead of drinking and I agree on them not having a chemistry analog, I think you can acutally use a Hardsuit Storage pod for decomtamination. I'm just testing it out now. Edit: Yes, yes you can. Feed them some burn meds before hand though. A naked human got 82 burn form it.

You've changed my mind on this PR. After testing it myself and reading other's comments I think it's a good step in the right direction. 

@Isy232 Hey just testing it out now and found that the pills in the kelo and bicard bottles are named salicylic acid 15u. They're the correct chems, just named wrongly

**Comment by Isy232** [comment]
Chessus, Thanks for finding that, I fixed it

**Comment by Trash0077** [comment]
So far so good, though i do not clearly understand if salicylic acid now heals 3 damage all time, or just as it was, only healing 3 damage when more than 50 damage? Well, regardless of that its a good change, though i definetly must look forward to a wiki update on those chems aswell.

**Comment by Isy232** [comment]
> So far so good, though i do not clearly understand if salicylic acid now heals 3 damage all time, or just as it was, only healing 3 damage when more than 50 damage? Well, regardless of that its a good change, though i definetly must look forward to a wiki update on those chems aswell.

Sali heals 3 brute per tick at all times.

**Comment by Rukofamicom** [comment]
> i definetly must look forward to a wiki update on those chems aswell.

I'll be the one updating it once there's full agreement on the effects and this is actually merged.

**Comment by LivingBatteryBox** [comment]
I feel that this change makes miners too reliant on medipens. I get nerfing styptic and silver sulf sprays to a degree, but making them metabolize significantly faster while healing less is a bit much. Miners who don't have 50 billion points to spend on medipens will be stuck with the shelter capsule's now worthless patches and sprays. Plus the stam damage is insult to injury. I can see the counter argument of _Git Gud_, but I don't think having this mentality on a server meant for new players is healthy. Not being able to pre-spray to deal with a tendril or a megafauna is really going to suck as a miner.

**Comment by Naevii** [comment]
Does synthflesh sprays still work after disabling splashing effect on synthflesh ? 
Will unhusking body with synthflesh still work ? 

I am confused with this changes, i understand this is needed, i think disabling splashing and giving minor stamina damage is a good change. However i am not sure if all of those chemicals needs changes (nerfs) but i think we won't know untill we test it in game. 

I still think medical requires rather full rework (i posted a few my ideas on pr-discussion on discord, search in pinned messages) than trying to balance this, but it is what it is.

**Comment by That0nePerson** [comment]
@Isy232 Why do we need a Corazone or Carthatoline nerf? I never even see them used in the first place, and their OD effects are so deadly for... no reason that I can see.

As a med main though, I love the rest of the changes.

**Comment by Chessus** [comment]
> Does synthflesh sprays still work after disabling splashing effect on synthflesh ?
> Will unhusking body with synthflesh still work ?

Sf Sprays don't work and are removed. Yes, unhusking works as you can make sf patches (would be quicker now as patches get it into the system faster)

**Comment by St0rmC4st3r** [comment]
> @Isy232 Why do we need a Corazone or Carthatoline nerf? I never even see them used in the first place, and their OD effects are so deadly for... no reason that I can see.
> 
> As a med main though, I love the rest of the changes.

Histamine is not as deadly as you think.

**Comment by yyzsong** [comment]
Histamine is literally in the syndicate uplink and metabolizes at 1/4th the rate of stuff like bicardine

**Comment by St0rmC4st3r** [comment]
> Histamine is literally in the syndicate uplink and metabolizes at 1/4th the rate of stuff like bicardine

But it does nearly nothing unless overdosed. Also finally a use for diphenhydramine.
If it turns out to be too lethal, I will make a PR that will create a temporary way to delay the effects.

**Comment by Chessus** [comment]
@Isy232 https://github.com/BeeStation/BeeStation-Hornet/blob/ea1cd3909abef70dc9d7a06a501d6a3c02c0b6db/code/modules/reagents/chemistry/reagents/medicine_reagents.dm#L100-L115

It could also give birth to this unused chem. Gives you minor poison but doesn't make you drowsy (Which has a chance to knock out) like straight diphenhydramine does. It's been there forever but has no recipe.

**Comment by Isy232** [comment]
> I feel that this change makes miners too reliant on medipens...

Well, sadly this is kind of unavoidable. Maybe certain lavaland monsters should be nerfed to compensate but in general reworking chems has such vast effects that this is sure to change the balance of things.

If this gets TM'ed or merged and mining is made too hard I'll make a follow up PR to nerf lavaland monsters.

I'd need to talk to someone who is more experienced with mining though, I barely play it.

___

> Does synthflesh sprays still work after disabling splashing effect on synthflesh ?
> Will unhusking body with synthflesh still work ?

Yes to both.
Sprays actually use the patch method of application so splashing changes won't affect them at all.

I originally thought they used splash method of application so thats why there is so much confusion.

```
/obj/item/reagent_containers/medspray
	name = "medical spray"
        (...)
	var/apply_type = PATCH
```
____

> @Isy232 Why do we need a Corazone or Carthatoline nerf? I never even see them used in the first place, and their OD effects are so deadly for... no reason that I can see.

Mostly to bring toxin healing medications in line with other chems. We're anticipating them to be used more now that the basic chems are made worse.

If these chems are never used like before they will be buffed again probably.

___

> @Isy232
> 
> https://github.com/BeeStation/BeeStation-Hornet/blob/ea1cd3909abef70dc9d7a06a501d6a3c02c0b6db/code/modules/reagents/chemistry/reagents/medicine_reagents.dm#L100-L115
> 
> It could also give birth to this unused chem. Gives you minor poison but doesn't make you drowsy (Which has a chance to knock out) like straight diphenhydramine does. It's been there forever but has no recipe.

That would be best fitted to another PR. Me and Ruko have been talking about maybe further changing chems adding a third tier beyond just Bicardine/Sali and their equivalents. This would be done later though.

**Comment by Rukofamicom** [comment]
> I feel that this change makes miners too reliant on medipens. I get nerfing styptic and silver sulf sprays to a degree, but making them metabolize significantly faster while healing less is a bit much. Miners who don't have 50 billion points to spend on medipens will be stuck with the shelter capsule's now worthless patches and sprays. 

As someone who plays both medbay and shaft miner as my primary departments I disagree with both the sentiment that shelter meds are useless now and also the idea that miners need access to an overabundance of healing without turning ores in to pay for medipens. 

While the patches are now much weaker, they're far from worthless unless you're just being totally careless and relying on them for constant healing. A shaft miner that isn't in critical condition can still fully heal off of a capsule without overdosing.

Shaft miners that want to just rush Megafauna being forced to actually mine enough points to pay for survival pens is good for everyone except the miner that wants to ignore their actual job entirely. 

While this is a side-effect I hadn't considered, I see it as 100% beneficial for the health of the game.

**Comment by ike709** [comment]
YOLO
